ui/pages/send/send-content/add-recipient/add-recipient.component.js

```diff
@@ -28,7 +28,7 @@ export default class AddRecipient extends Component {
     isUsingMyAccountsForRecipientSearch: PropTypes.bool,
     recipient: PropTypes.shape({
       address: PropTypes.string,
-      nickname: PropTypes.nickname,
+      nickname: PropTypes.string,
```

Comment:
Noticed this invalid `PropType` was causing:
<img width="582" alt="Screen Shot 2022-09-18 at 8 45 55 PM" src="https://user-images.githubusercontent.com/8732757/190955635-cd5a04d6-5b2a-43af-8328-8b8b3dfe2d0a.png">